bin/resolve-dependencies

```diff
@@ -72,6 +78,17 @@ async function resolve(specifier) {
   return {
     name: package.name,
     version: package.version,
-    export: (package.unpkg || package.jsdelivr || package.browser || package.main).replace(/^\.\//, "")
+    export: main(package)
```

Comment:
This is a workaround for JSZip’s use of the nonstandard `browser` object for Browserify:

https://github.com/Stuk/jszip/blob/e5b3f0ddaa8182cd6ea253e97f678b9f36d0d8ac/package.json#L27-L30

I modeled this code after the same code from d3-require.